eth/tracers/native/noop.go

```diff
@@ -0,0 +1,46 @@
+package native
+
+import (
+	"encoding/json"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/eth/tracers"
+)
+
+func init() {
+	tracers.RegisterNativeTracer("noopTracerNative", NewNoopTracer)
+}
+
+type noopTracer struct{}
+
+func NewNoopTracer() tracers.Tracer {
+	return &noopTracer{}
+}
+
+func (t *noopTracer) CaptureStart(env *vm.EVM, from common.Address, to common.Address, create bool, input []byte, gas uint64, value *big.Int) {
+}
+
+func (t *noopTracer) CaptureEnd(output []byte, gasUsed uint64, _ time.Duration, err error) {
+}
+
+func (t *noopTracer) CaptureState(env *vm.EVM, pc uint64, op vm.OpCode, gas, cost uint64, scope *vm.ScopeContext, rData []byte, depth int, err error) {
+}
```

Comment:
Let's remove this ?

Reply:
We can't really otherwise it wouldn't implement the `Tracer` interface. Shouldn't make a big difference I hope

Reply:
Yeah I know, I thought about it after I wrote this comment and went "Doh!". Was meaning to go back and delete it, guess I'll have to live with the shame now :)